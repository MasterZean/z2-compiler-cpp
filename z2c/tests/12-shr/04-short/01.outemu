Short{4}
Short{4}
Short{4}
Short{4}
Short{4}
Short{4}
Short{4}
Short{4}
Short{4}
