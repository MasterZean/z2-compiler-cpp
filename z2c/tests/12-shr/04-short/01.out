70 >> 4;
70 >> 4u;
70 >> 4;
70 >> 4u;
70 >> 4;
70 >> 4u;
70 >> 4ll;
70 >> 4ull;
70 >> 4u;
DummyClass(1, 7): error:
	can't apply operator '>>' on types: 
		'Short' and 
		'Bool'
DummyClass(2, 7): error:
	can't apply operator '>>' on types: 
		'Short' and 
		'Bool'
DummyClass(11, 7): error:
	can't apply operator '>>' on types: 
		'Short' and 
		'Float'
DummyClass(12, 7): error:
	can't apply operator '>>' on types: 
		'Short' and 
		'Double'
DummyClass(13, 7): error:
	can't apply operator '>>' on types: 
		'Short' and 
		'Char'
